README.md

```diff
@@ -46,6 +46,48 @@ worker-ubuntu16-04_1  | xxxx/xx/xx xx:xx:xx rpc error: code = Unavailable desc =
 
 This is usually because container of the worker has started before the scheduler and so it cannot connect. After a second or so, this error message should stop.
 
+## Platform properties
+
+For clients to be able to use Buildbarn, clients need to send actions with platform properties that
+match the platform properties that Buildbarn workers register themselves with.
+
+For Bazel clients, platform properties can be set in the `.bazelrc` or as command line parameters:
+
+```
+--remote_default_exec_properties=OSFamily=Linux
+--remote_default_exec_properties=container-image=docker://marketplace.gcr.io/google/rbe-ubuntu16-04@sha256:b516a2d69537cb40a7c6a7d92d0008abb29fba8725243772bdaf2c83f1be2272
+```
+
+Buildbarn workers are configured with its associated `.jsonnet` configuration file.
+
+```
+{
+  # See worker-ubuntu16-04.jsonnet for the full worker configuration.
+  buildDirectories: [{
+    native: {
```

Comment:
Maybe trim down this configuration further, so that only the platform properties remain?

Reply:
Make sense. Let me know if the following trimming works for you.